packages/document/docs/zh/config/output/inline-styles.mdx

```diff
@@ -86,7 +86,7 @@ export default {
 export default {
   output: {
     inlineStyles({ size }) {
-      return size < 10 * 1000;
+      return size < 10 * 1024;
```

Comment:
Can we change `10KB` to `10kB` (also the English document), so we do not need to update the example